wcivf/apps/elections/tests/test_election_views.py

```diff
@@ -256,7 +256,8 @@ def test_no_candidates_turn_up(self):
         )
         self.assertContains(
             response,
-            "This election will not take place",
+            f"This election will not take place because <a href='{self.post_election.ynr_link}'>no candidates have been nominated to stand</a>.",
```

Comment:
I'm struggling to find an example of this type of assert which checks for the presence of a url. What am I missing here?